schema/sale_artwork.js

```diff
@@ -33,6 +40,17 @@ const SaleArtworkType = new GraphQLObjectType({
       bidder_positions_count: {
         type: GraphQLInt,
       },
+      bidder_positions: {
+        type: new GraphQLList(BidderPosition.type),
+        resolve: (sale_artwork, options, { rootValue: { accessToken } }) => {
+          return gravity.with(accessToken)('me/bidder_positions', {
+            artwork_id: sale_artwork.id,
+            sale_id: sale_artwork.sale_id,
+          })
+            .then(response => enhance(response, { sale_artwork }))
```

Comment:
We're going to need access to the `sale_artwork` for a few different fields that I haven't implemented yet